src/sentry/tasks/reprocessing.py

```diff
@@ -61,6 +61,8 @@ def clear_expired_raw_events():
     from sentry.models.rawevent import RawEvent
     from sentry.models.reprocessingreport import ReprocessingReport
 
+    MAX_BATCHES_PER_MODEL = 1000
```

Comment:
Does this mean the script won't remove everything in one go then?

Reply:
Yeah, but it runs every 15 minutes. It's 1000 batches of 1000 deletions, so 1 milliion every 15 minutes. The `sentry_rawevent` table has around 50m events, so even that will only take a short while to clean up.

Also I'm going to wait until next week to merge this so it's not running over the weekend

Reply:
👍 

Will let processing team look at this as well. Looks good though